integration-tests/azure/README.adoc

```diff
@@ -0,0 +1,16 @@
+== Camel Quarkus Azure (deprecated client v8) integration tests
+
+By default the integration tests run against a local https://github.com/Azure/Azurite[Azurite] container.
+
+To run the tests against the real remote Azure API, you need the following:
+
+* An Azure https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-create-account-block-blob?tabs=azure-portal[Storage Account]
```

Comment:
@ppalaga I'm missing instructions for the Queue Storage.

Reply:
There are no specific instructions for queue. The Storage account holds for both. Maybe we should rephrase the readme?

Reply:
That is weird, i've followed the readme and only the `testAzureBlobComponent` is passing :(

Reply:
I have it ! One should follow https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&tabs=azure-portal as it contains also the Queue service.

Reply:
Thanks, fixed in 58859a2